src/haiku/meson.build

```diff
@@ -1,11 +1,11 @@
-rc = meson.find_program('rc')
-xres = meson.find_program('xres')
+rc = find_program('rc')
+xres = find_program('xres')
 
 rsrc = custom_target(
   'mpd.rsrc',
   output: 'mpd.rsrc',
   input: 'mpd.rdef',
-  command: [rc, '-o', '@OUTPUT@', '@INPUT@'],
+  command: [rc, '-o -q', '@OUTPUT@', '@INPUT@'],
```

Comment:
What's this? Unrelated? Does this even work, adding two options separated by a space in one argument?